dev/com.ibm.ws.microprofile.metrics.cdi.2.0/bnd.bnd

```diff
@@ -55,6 +55,7 @@ Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
 	com.ibm.ws.kernel.service;version=latest,\
 	com.ibm.ws.microprofile.metrics.2.0;version=latest,\
 	com.ibm.websphere.org.eclipse.microprofile.metrics.2.0;version=latest,\
+	com.ibm.websphere.org.eclipse.microprofile.metrics.2.2;version=latest,\
```

Comment:
On the buildpath you should only pull in the lowest version of the api that you want to be compatible with. So either this new line should be removed or the 2.0 line above should be removed.